### PR TITLE
Copy actual to expected script

### PR DESCRIPTION
**Pull request by jcivlin**
Scripts copies all .actual to .expected files in subdirectories of the directory given in input parameter.

### CODE REVIEW

File: language/tools/move-mv-llvm-compiler/utils/copy_actual_to_expected.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright (c) The Diem Core Contributors
+# Copyright (c) The Move Contributors
+# SPDX-License-Identifier: Apache-2.0
+
```
Comment (ksolana) [inline language/tools/move-mv-llvm-compiler/utils/copy_actual_to_expected.sh:6]: It'll be nice to document what this script does.
Let's add a sample use case for someone who doesn't know how to use it.

**Comment by dmakarov** [comment]
why push every script you use for your convenience to repository?

**Comment by dmakarov** [comment]
This script can be replaced by a single line shell command. I would prefer the repository doesn’t include shell commands wrapped in scripts, polluting the filesystem.

**Comment by ksolana** [comment]
closing this after discussing offline.

**Pull request closed without merging**